src/BizHawk.Client.Common/Api/Classes/MemoryApi.cs

```diff
@@ -303,7 +305,7 @@ public void WriteFloat(long addr, float value, string domain = null)
 				LogCallback($"Warning: Attempted write {addr} outside memory size of {d.Size}");
 				return;
 			}
-			d.PokeUint(addr, BitConverter.ToUInt32(BitConverter.GetBytes(value), 0), _isBigEndian);
+			d.PokeUint(addr, Unsafe.As<float, uint>(ref value), _isBigEndian);
```

Comment:
Is there a reason you didn't make this into a global common function (like [before](https://github.com/TASEmulators/BizHawk/commit/0f22ef2ad05fe8f72f06fcb2b6a932008eba69a4#diff-cf06fd3b2597d6a73503739b41c844d1e0738f241b69f19de4e9865629e82a0fL160))?

Reply:
I realised that having an allowlist of helpers wouldn't prevent you from using `Unsafe.As` directly and typo-ing the type arguments, so I'd have to use an Analyzer, and then there's no benefit to those explicit helpers.

Reply:
It doesn't prevent developers from using it, no, I just thought it might be practical as this uint<->float reinterpreting seems to be somewhat common around the watch area and having our own specific function for it could prevent someone using something like `BitConverter` instead.

Reply:
Resolved with https://github.com/TASEmulators/BizHawk/commit/10ebea4dda87cbc65dd3dd74cd83655b1a69dff1 (except in Cores for now)